esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3165,7 +3163,7 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
         bool serverFound = false;
         unsigned filesFound = 0;
         IArrayOf<IEspPhysicalFileStruct> files;
-        bool isIPAddressReq = isIPAddress(dropZoneServerReq);
+        bool isIPAddressReq = !isEmptyString(dropZoneServerReq) && isIPAddress(dropZoneServerReq);
```

Comment:
I think isIPAddress already checks for empty string?

Reply:
The duplicated string check will be removed.